spec/features/shared/header_spec.rb

```diff
@@ -94,6 +99,15 @@
       end
     end
 
+    context 'clicking on the Community link' do
+      it 'should redirect to XR-Network index page' do
+        click_on 'XR Network'
+        click_on 'Community'
+        expect(page).to have_current_path('/communities/xr-network')
+      end
+    end
+
+
```

Comment:
I think we just need a test for the other pages now. This one looks great!

Reply:
Added tests for all pages.